src/main/resources/application.yml

```diff
@@ -10,11 +10,18 @@ queueconfig:
   action-field-queue: Action.Field
   outbound-exchange: adapter-outbound-exchange
   adapter-outbound-queue: RM.Field
+  receipt-queue: FieldWorkAdapter.Responses
+  receipt-routing-key: Receipt.Responses.binding
```

Comment:
Might this need to be `Case.Responses.binding` to match the binding name in pubsub?

Reply:
I reckon pubsub should be changed then as this is correct (I did pubsub wrong, soz)

Reply:
Looks like the event dictionary says this needs to be called `event.response.receipt`?

Reply:
I thought the topic was called events?

Reply:
ok me wrong